example_project/example_project/management/commands/populate_redis.py

```
@@ -283,6 +285,43 @@ def create_test_data(self, redis_conn, db_num, key_count=100, options=None):
+                },
+            }
+            try:
+                redis_conn.json().set(json_key, "$", json_value)
+                json_keys_created += 1
+                created_keys += 1
+            except Exception as e:
+                # RedisJSON module not available, skip JSON keys
+                if "unknown command" in str(e).lower() or "json" in str(e).lower():
+                    self.stdout.write(
+                        self.style.WARNING(
+                            f"      RedisJSON module not available, skipping JSON keys"
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)